tests/admin_filters/tests.py

```diff
@@ -381,6 +383,11 @@ def test_datefieldlistfilter(self):
             )
         )
 
+    @unittest.skipIf(
+        sys.platform.startswith('win'),
+        "Windows doesn't support setting a timezone that differs from the "
+        "system timezone."
+    )
```

Comment:
I was thinking we could only skip this test if the system timezone wasn't equal to `settings.TIMEZONE` but it looks like it would require a lot workaround to get things working correctly.